bootstrap/reset_db.py

```
@@ -0,0 +1,38 @@
+    try:
+        async with engine.begin() as conn:
+            print("Dropping public schema cascade...")
+            await conn.execute(text("DROP SCHEMA public CASCADE"))
+            print("Recreating public schema...")
+            await conn.execute(text("CREATE SCHEMA public"))
+            print("Granting privileges on public schema...")
+            await conn.execute(text("GRANT ALL ON SCHEMA public TO public"))
+        
+        print("\nDatabase has been completely wiped!")
+        print("You can now run 'bootstrap.cmd' to recreate tables and reseed data.")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)